pages/home/auth-providers/oauth2.mdx

```diff
@@ -84,7 +84,7 @@ Each service expects a slightly different set of parameters in the `oauth2` sect
 
 The only supported OAuth 2.0 flow is the authorization code grant (with or without PKCE).
 
-`scope_delimiter` _(optional, default `" "` / space)_: The delimiter for scopes, if any.
+`scope_delimiter` _(optional, defaults to the space character)_: The delimiter for scopes, if any.
```

Comment:
```suggestion
`scope_delimiter` _(optional, defaults to the space character)_: The delimiter for scopes.
```

There must be one. It just has a default

Reply:
Sorry, my first comment wasn't clear. The "if any" was confusing me when I read it. I saw it had a default, but the "if any" implied I could somehow specify there should be no delimiter. For clarity, I think we should remove the "if any"

Reply:
Ah 👍 Makes sense.